webapp/src/js/services/live-list.js

```diff
@@ -40,6 +39,8 @@ angular.module('inboxServices').factory('LiveListConfig',
     // Configure LiveList service
     return function($scope) {
 
+      const HARDCODED_CONTACT_TYPES = ['district_hospital', 'health_center', 'clinic', 'person'];
```

Comment:
Maybe we can reuse `ContacTypes > HARDCODED_CONTACT_TYPES` ?

Reply:
Yeah good idea. I've exposed it from ContactTypes and used it here.